### PR TITLE
Storey: Introduction

### PR DESCRIPTION
**Pull request by uint**

### CODE REVIEW

File: src/pages/storey.mdx [hunk 1]
```diff
@@ -3,3 +3,24 @@ tags: ["storey"]
...
+software does.
+
+While StoragePlus has been around for years and is the established way of
+storing your data on-chain, Storey is the new kid on the block.
+
+Storey is currently **prototype software**, meaning the API might change
+rapidly, **storage layout might break between versions**, and you might stumble
+into bugs and missing features. It's probably a good idea not to use it for
+serious production software just yet. If you do, please keep in mind the
+increased maintenance burden and be careful!
```
Comment (aumetra) [inline src/pages/storey.mdx:23]: Should this maybe be in a call-out block to make it more visually distinct?

File: src/pages/storey.mdx [hunk 2]
```diff
@@ -3,3 +3,24 @@ tags: ["storey"]
 ---
 
 # Introduction
+
+[Storey](https://docs.rs/cw-storey/latest/cw_storey/) is an alternative to
```
Comment (aumetra) [inline src/pages/storey.mdx:9]: Have we ever settled on whether we put links inline or split them up with the actual URLs at the bottom of the doc?

Comment (uint) [reply src/pages/storey.mdx:9]: I don't know, but what's the worry? That we'll have an increased maintenance burden if we're not consistent with this?

File: src/pages/storey.mdx (hunk 1 above)
Comment (uint) [reply src/pages/storey.mdx:23]: Yeah, good call

File: src/pages/storey.mdx (hunk 2 above)
Comment (aumetra) [reply src/pages/storey.mdx:9]: No, no worries, just thought out loud about consistency

**Review by aumetra — APPROVED** [review]

**Pull request merged**